### PR TITLE
[ADD][12.0] portal_sale_mail_link: New module

### PR DESCRIPTION
**Pull request by chienandalu**
This module injects a button in the sale confirmation email with a portal link
that the customer can follow in order to sign, pay or just review the order like
it used to be in <=v11.

image

cc @Tecnativa TT25746

### CODE REVIEW

**Review by pedrobaeza — COMMENTED** [review]
Is it possible to develop a test exercising this?

**Comment by chienandalu** [comment]
Yes, I think we can use the preview wizard

**Comment by chienandalu** [comment]
Tests added

File: portal_sale_mail_link/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: portal_sale_mail_link
+#: code:addons/portal_sale_mail_link/models/mail_template.py:21
+#, python-format
+msgid "\n"
```
Comment (ernestotejeda) [inline portal_sale_mail_link/i18n/es.po:21]: It should be the translation, not the original text

**Comment by pedrobaeza** [comment]
@chienandalu please attend comment and check Travis result.

File: portal_sale_mail_link/i18n/es.po (hunk 1 above)
Comment (chienandalu) [reply portal_sale_mail_link/i18n/es.po:21]: Thanks @ernestotejeda :)

**Comment by chienandalu** [comment]
fixed, I think the travis error was a random one

**Review by victoralmau — APPROVED** [review]

**Review by ernestotejeda — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@chienandalu is this finally needed?

**Comment by chienandalu** [comment]
I'm afraid it isn't as after all there's a wrapper that already injects the button depending on these conditions. I close it

**Pull request closed without merging**